### PR TITLE
'suneel branch'

### PR DESCRIPTION
**Pull request by suneel-katta**
testing

### CODE REVIEW

File: sample.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This is suneel
+This is phani
```
Comment (phanindra-kumar-pedipini) [inline sample.txt]: Please modify phanindra kumar instead of phani

File: sample.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+This is suneel
```
Comment (phanindra-kumar-pedipini) [inline sample.txt]: Please enter full name

**Review by phanindra-kumar-pedipini — CHANGES_REQUESTED** [review]

File: sample.txt (hunk 1 above)
Comment (suneel-katta) [reply sample.txt]: yes modified

File: sample.txt (hunk 2 above)
Comment (suneel-katta) [reply sample.txt]: modified

**Pull request merged**